endpointmanager/pkg/endpointmanager/postgresql/fhirendpointstore.go

```diff
@@ -2,13 +2,21 @@ package postgresql
 
 import (
 	"context"
+	"database/sql"
 	"encoding/json"
 
 	"github.com/lib/pq"
 	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/capabilityparser"
 	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager"
 )
 
+// prepared statements are left open to be used throughout the execution of the application
+// TODO: figure out if there's a better way to manage this for bulk calls
+var areFHIREndpointStatementsPrepared = false
```

Comment:
Same logic described below about relying on package initialization to assign these variables their values so that `areFHIREndpointStatementsPrepared` will not be needed and `prepareFHIREndpointStatements` will not need to be called all over the place